src/bricks/transformers/brickFactory.ts

```
@@ -255,7 +257,7 @@ class UserDefinedBrick extends BrickABC {
     const pipeline = castArray(this.component.pipeline);
     const last = pipeline.at(-1);
 
-    if (this.id === last?.id) {
+    if (!last?.id || this.id === last.id) {
```

In what situation would id be null if id is present? Isn't id non-null? https://github.com/pixiebrix/pixiebrix-extension/blob/f27fd0320157af735ef87c7ac739a5087aa07b24/src/bricks/types.ts#L122-L122

 If we think that id shouldn't be null, I'd recommend we:

- Check for last not being null
- Add an assertion that `last.id` is not null

My general philosophy is we want to fail fast vs. having to track down bad values